src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala

```diff
@@ -454,15 +455,20 @@ object TezosDatabaseOperations extends LazyLogging {
     predicates: List[Predicate],
     ordering: List[QueryOrdering],
     aggregation: List[Aggregation] = List.empty,
+    outputType: OutputType,
```

Comment:
should we expect a default of Json here?

Reply:
Its being done during the transformation from `ApiQuery` to `Query`